server/Gemfile

```diff
@@ -0,0 +1,7 @@
+source 'https://rubygems.org'
+
+ruby '~> 3.1.4'
+
+gem 'roda'
+
+gem 'thin', require: false # Use Thin as the webserver
```

Comment:
I think for the simplicy of use it should be same, one Gemfile for whole repository